src/index.ts

```diff
@@ -18,11 +18,11 @@ export type PluginOptions = {
   tsconfig?: string | false
 } & Pick<SwcConfig, Exclude<keyof SwcConfig, 'filename'>>;
 
-const INCLUDE_REGEXP = /\.[jt]sx?$/;
+const INCLUDE_REGEXP = /\.m?[jt]sx?$/;
 const EXCLUDE_REGEXP = /node_modules/;
 
 const resolveFile = (resolved: string, index = false) => {
-  for (const ext of ['.ts', '.js', '.tsx', '.jsx']) {
+  for (const ext of ['.ts', '.js', '.tsx', '.jsx', '.mjs']) {
```

Comment:
```suggestion
  for (const ext of ['.ts', '.mjs', '.js', '.tsx', '.jsx']) {
```

`.mjs` should have higher priority than `.js`.

Reply:
Fixed it in a separate pull: https://github.com/SukkaW/rollup-plugin-swc/pull/3